docs/conf.py

```diff
@@ -0,0 +1,60 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "pynntp"
+version = "2.0.1"
+copyright = "2024, Byron Platt"  # noqa: A001
+author = "Byron Platt and contributors"
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+autoapi_dirs = ["nntp"]
+exclude_patterns = [
+    ".*/*",
+    ".DS_Store",
+    "docs/_build",
+    "LICENSE.rst",
+    "Thumbs.db",
+]
+extensions = [
+    "autoapi.extension",
+    "myst_parser",
+]
+myst_enable_extensions = [
+    "amsmath",
+    "attrs_inline",
+    "colon_fence",
+    "deflist",
+    "dollarmath",
+    "fieldlist",
+    "html_admonition",
+    "html_image",
+    # "linkify",
+    "replacements",
+    "smartquotes",
+    "strikethrough",
+    "substitution",
+    "tasklist",
+]
+myst_fence_as_directive = [
+    "include",
+]
+source_suffix = {
+    ".rst": "restructuredtext",
+    ".txt": "markdown",
+    ".md": "markdown",
+}
+suppress_warnings = ["epub.unknown_project_files"]
+templates_path = ["_templates"]
+
+# -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+
+html_static_path = ["_static"]
+html_theme = "alabaster"
```

Comment:
Not using `Furo` despite installing the dependency?

Reply:
Good catch!  Which theme would you prefer?

Reply:
Good question. I actually quite like `sphinx-book-theme` https://sphinx-themes.org/sample-sites/sphinx-book-theme/

Reply:
I retract that statment. I just had a play with book and it wasn't what I expected.

Reply:
Done.